src/lib/installation/clients/inst_complex_welcome.rb

```diff
@@ -68,6 +68,9 @@ def main
 
       Yast::Wizard.EnableAbortButton
 
+      # Preselect the product if there is only one so the license can be shown.
+      products.first.select if products.size == 1 && !products.first.selected?
```

Comment:
Selecting an already selected product should be safe, shouldn't be?

Or any reason why not use a simple
```suggestion
      products.first.select if products.size == 1
```
here?

Reply:
Well, initializing a product from the full medium might have some side effects: https://github.com/yast/yast-packager/blob/597a6a5b80ebbeec91c4b256457a0ea1f5a3b3ea/src/lib/y2packager/repo_product_spec.rb#L70-L85. So I preferred to be more conservative here.

Actually, it brings me to this bug I have just opened: https://bugzilla.suse.com/show_bug.cgi?id=1193375.